py-utils/src/utils/msg_bus/benchmarks/kafka_python_vs_confluent.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import absolute_import

```

Comment:
Move this code to cortx-experiments repo

Reply:
moved